src/app.ts

```diff
@@ -13,14 +14,60 @@ interface TemperatureSummary {
   average: number
 }
 
+const temperatures: TemperatureReading[] = []
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  temperatures.push(...readings)
 }
 
 export function getTemperatureSummary(
```

Comment:
one of the problems with your implementation is that every time we called this method, the information is calculated again. The idea of `processReadings` is to process the information once and be accessible every time you need it but without recalculating everything again.